packages/trader/src/App/Components/Elements/market-countdown-timer.jsx

```diff
@@ -89,6 +89,7 @@ const MarketCountdownTimer = ({ active_symbols, is_main_page, setIsTimerLoading,
 
             whenMarketOpens(0, symbol);
         }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Some of the ignored warnings is because eslint ask to get lots of dependency and this results in lots of unnecessary rerendering. for example for `[items]` dependency the eslint ask for adding the `props` to dependency array and since we have lots of props, I do not think it's  a good idea. For `[location]` eslint ask to remove it since it's unnecessary, but we need to rerender page when the location is changed.